SQL/Release_patches/24.1_To_25.0_upgrade.sql

```diff
@@ -498,7 +451,8 @@ INSERT INTO ConfigSettings
     Name="gui";
 
 INSERT INTO Config (ConfigID, Value) SELECT ID, 'false' FROM ConfigSettings WHERE Name="useEEGBrowserVisualizationComponents";
-CREATE INDEX `i_violations_resolved_extid_type` ON `violations_resolved` (`ExtID`, `TypeTable`);
+
```

Comment:
I don't think we should be changing this. Was there a problem with the order that it was, even if it wasn't chronological? That's the order that the patch was in when it was tested.

Reply:
we don't know.... we were disabling foreign keys in the same patch